src/openforms/submissions/models/submission_value_variable.py

```diff
@@ -358,28 +358,23 @@ def to_python(self) -> Any:
                 return aware_date.date()
 
             maybe_naive_datetime = parse_datetime(self.value)
-            if maybe_naive_datetime is not None:
-                if timezone.is_aware(maybe_naive_datetime):
-                    return maybe_naive_datetime.date()
-                return timezone.make_aware(maybe_naive_datetime).date()
+            if maybe_naive_datetime is None:
+                return
```

Comment:
(don't change this, but take it into account for the future)

generally, if we're deliberately returning `None` I prefer to make that explicit: `return None`, while a bare `return` is more intended as "stop with whatever we're doing and we don't particularly care about the exact return value".

The end result/return value is identical, but there's some different meaning/implication in the exact statement used.